lib/content-services/src/lib/category/categories-management/categories-management.component.spec.ts

```diff
@@ -167,44 +167,38 @@ describe('CategoriesManagementComponent', () => {
             });
 
-            it('should have correct label and hide button', () => {
-                const categoryControlLabel = fixture.debugElement.query(By.css('#adf-category-name-input-label')).nativeElement;
-                const categoryControlHideBtn: HTMLButtonElement = fixture.debugElement.query(By.css('.adf-category-name-field button')).nativeElement;
-                expect(categoryControlHideBtn).toBeTruthy();
-                expect(categoryControlHideBtn.attributes.getNamedItem('title').textContent.trim()).toBe('CATEGORIES_MANAGEMENT.HIDE_INPUT');
-                expect(categoryControlLabel.textContent.trim()).toBe('CATEGORIES_MANAGEMENT.NAME');
+            it('should return true when categoryNameControlVisible is true', () => {
+                component.categoryNameControlVisible = true;
+                const result = component.isNameCategoryVisible;
+                expect(result).toBeTrue();
             });
 
-            it('should hide and clear category control and existing categories panel on clicking hide button', fakeAsync(() => {
-                typeCategory('test');
-                const categoryControlHideBtn: HTMLButtonElement = fixture.debugElement.query(By.css('.adf-category-name-field button')).nativeElement;
-                const controlVisibilityChangeSpy = spyOn(component.categoryNameControlVisibleChange, 'emit').and.callThrough();
-                categoryControlHideBtn.click();
-                fixture.detectChanges();
+            it('should return false when categoryNameControlVisible is false and categories length is 0', () => {
+                component.categoryNameControlVisible = false;
+                component.categories = [];
+                const result = component.isNameCategoryVisible;
+                expect(result).toBeFalse();
+            });
+        });
 
-                const categoryControl: HTMLDivElement = fixture.debugElement.query(By.css('.adf-category-name-field')).nativeElement;
-                expect(categoryControl.hidden).toBeTrue();
-                expect(component.categoryNameControlVisible).toBeFalse();
-                expect(component.existingCategoriesPanelVisible).toBeFalse();
-                expect(controlVisibilityChangeSpy).toHaveBeenCalledOnceWith(false);
+        describe('isCategoryEmpty', () => {
+            it('should return true when categories is not empty', () => {
+                component.categories = [category3, category4];
+                const result = component.isCategoryEmpty;
+                expect(result).toBeTrue();
+            });
 
-                component.categoryNameControlVisible = true;
-                fixture.detectChanges();
-                tick(100);
-                expect(getCategoryControlInput().value).toBe('');
-            }));
+            it('should return false when categories is empty', () => {
+                component.categories = [];
+                const result = component.isCategoryEmpty;
```

Comment:
wrong test case... "isEmptyCategory" returning false? "is empty" for the empty categories should return true